### DISCUSSION TITLE
after alpha

### DISCUSSION THREAD

**Discussion by Earthegodlybeing** [SycNav/New-Crescent#11] [General]
after we finish zone 9 it will move to beta and we will continue to zone 18 and far more the content still in alpha everything may or may not change in the future please stand by!!!!

**Comment by Earthegodlybeing** [comment]
read it

**Reply by Earthegodlybeing** [reply]
im the mapper tho :)))))))))))))))))))))))))))))))))))))))))))))))))))))

**Comment by cryptalmist** [comment]
bro just stfu 💀
